src/rdkv/impl.c

```diff
@@ -53,6 +53,9 @@
 #define WEBCFG_PARTNER_JSON_FILE "/etc/partners_defaults_webcfg_video.json"
 #define WEBCFG_DB_STORE          "/opt/.webconfig.json"
 
+#define DEF_WEB_URL "https://cpe-config.xdp.comcast.net/api/v1/device/{mac}/config"
```

Comment:
Hard coding server urls in opensource is not ideally proper due to security reasons, or can this be read from a local tmp file ? Please check with security team and confirm to address default urls in a better way if possible..

Cc @shilpa24balaji

Reply:
Removed the hardcoding/macro and handled accordingly.